WasatchNET/Spectrometer.cs

```diff
@@ -3125,8 +3336,11 @@ public virtual async Task<double[]> getSpectrumAsync(bool forceNew=false)
                     sum[px] /= scanAveraging_;
             }
 
+            // This should come BEFORE bin2x2
             correctBadPixels(ref sum);
 
+            //correctBin2x2(ref sum);

```

Comment:
I see that you've removed this from getSpectrumAsync...where did it get moved to?

Reply:
For my ease of mind, could you confirm we're still applying 2x2 binning on appropriately configured IMX385 spectrometers?

Reply:
bin2x2 correction has been moved to line 3623